pkg/datapath/loader/cache.go

```diff
@@ -65,6 +65,13 @@ var ignoredELFPrefixes = []string{
 	"from-host",                  // Prog name
 	"to-netdev",                  // Prog name
 	"to-host",                    // Prog name
+	".BTF",                       // Debug
+	".BTF.ext",                   // Debug
+	".debug_ranges",              // Debug
+	".debug_info",                // Debug
+	".debug_line",                // Debug
+	".debug_frame",               // Debug
+	".debug_loc",                 // Debug
```

Comment:
The commit description says `as the iproute2 version shipped with Cilium is now able to parse BTF emitted by clang/llvm 10.0.0`. Could you elaborate what you mean by 'now'? It's still going through https://github.com/cilium/iproute2/commit/f823f36012fb5ab4ddfca6ed4ff56188730f281e no?

Reply:
On which kernels did we test? Is there a fall-back that when a kernel doesn't accept the BTF that we sanitise it on load?

Reply:
> Could you elaborate what you mean by 'now'? It's still going through cilium/iproute2@f823f36 no?

I was referring to the versions we ship with the docker images:
* https://github.com/cilium/iproute2/commit/2f278887268d60fdaed3c138d779827d9c0007b4 (see [here](https://github.com/cilium/image-tools/blob/9a8bfaa6c36fb0471bb0c3484f69eefeca0bafa2/images/iproute2/checkout-iproute2.sh#L11))
* LLVM 10.0.0 (see [here](https://github.com/cilium/image-tools/blob/9a8bfaa6c36fb0471bb0c3484f69eefeca0bafa2/images/llvm/checkout-llvm.sh#L11))

> On which kernels did we test? 

So far I run a connectivity check on all CI kernels, so 4.9, 4.19, 5.4 (GKE) and net-next

> Is there a fall-back that when a kernel doesn't accept the BTF that we sanitise it on load?

There isn't as initially I assumed having debug symbols compiled in would be an opt in feature rather than something that is on by default